### DISCUSSION TITLE
How do i get the flyouts to work with speaker controls

### DISCUSSION THREAD

**Discussion by JDelii** [ShankarBUS/ModernFlyouts#565] [General]
I downloaded Modern Flyouts (MS Store) recently and they seem to work fine for everything else. but for some reason they don't appear when controlling the volume on my speakers. Instead of the modern flyouts flyout it shows the normal windows 10 one. But the modern flyouts one shows up when doing the FN + (Volume control) keys. is this something that can be fixed?

**Comment by prakharb5** [comment]
Try the new 0.9.x beta. We recently fixed this bug there :)

For further reports, please use the issues page, not discussions.

**Reply by JDelii** [reply]
Oh my bad, didn't know there was a specific spot for that. I'm new to github lol
